templates/components/cart/content.html

```diff
@@ -85,7 +85,7 @@ <h2 class="cart-item-name">
                 </td>
                 <td class="cart-item-block cart-item-info">
                     <span class="cart-item-label">{{lang 'cart.checkout.price'}}</span>
-                    {{#or ../customer (if ../theme_settings.restrict_to_login '!==' true)}}
+                    {{#or ../customer (unless ../settings.hide_price_from_guests) (if type '==' 'GiftCertificate')}}
```

Comment:
@bc-yevhenii-buliuk This is because GiftCertificate prices are not set by the merchant, they are set by the customer when adding the Gift Certificate to the cart. So although we may be hiding product prices, there is no need to hide the Price for a gift certificate since it is already known by the customer and also included in the product name.

Without this condition, the cart item would look like this:

| Item  | Price | Quantity | Total |
| - | - | - | - |
| **$50 Gift Certificate**  | Login for pricing  | 1 | -- |

It would not make sense to show "login for pricing" for gift certificates if the price is in the name :-)

**With** the condition, the cart looks like this which looks better:
![image](https://user-images.githubusercontent.com/19815878/190694574-d8d92b71-e925-4727-abed-342d4768dc5b.png)

Reply:
@jordanarldt make sense. Thanks for the detailed explanation!

Reply:
@bc-yevhenii-buliuk Sure thing! Marking this as resolved